crates/auxmos/src/gas/constants.rs

```diff
@@ -82,7 +82,7 @@ pub const HEAT_CAPACITY_VACUUM: f32 = 7000.0;
 /// The Stefan-Boltzmann constant. M T^-3 Θ^-4
 pub const STEFAN_BOLTZMANN_CONSTANT: f64 = 5.670_373e-08; // watts/(meter^2*kelvin^4)
 
-const SPACE_TEMP: f64 = T20C as f64;
+const SPACE_TEMP: f64 = TCMB as f64;
```

Comment:
gonna repeat that this is entirely for gameplay reasons--every single turf is a blackbody radiator, so having this on and turning on heat processing will cause every single tile to not-that-rapidly cool to the CMB unless there's something there to explicitly counteract it

Reply:
it appears this isn't even used anymore so this is fine actually